frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1538,6 +1538,12 @@ export const surveyLogic = kea<surveyLogicType>([
                             sanitizedAppearance.submitButtonTextColor,
                             'button text color'
                         ),
+                        widgetSelector:
+                            type === 'widget' &&
```

Comment:
@lucasheriques 
this should be using the `SurveyType` type and `widgetType` could be an enum as well